packages/services/src/block_committer.rs

```diff
@@ -228,6 +228,13 @@ mod tests {
         async fn balance(&self) -> ports::l1::Result<U256> {
             self.api.balance().await
         }
+
+        async fn get_transaction_response(
+            &self,
+            _tx_hash: [u8; 32],
+        ) -> ports::l1::Result<Option<TransactionResponse>> {
+            Ok(None)
+        }
```

Comment:
Why is this function a no-op at the moment?

Reply:
This is a mock in one of the tests. Because the test do not use/need this method it is just a no-op.